src/ui/window/BrowserWindow.qml

```diff
@@ -185,6 +186,13 @@ ApplicationWindow {
                 ]
                 rightActions: [
                     Action {
+                        iconName: window.isFullScreen ? "navigation/fullscreen_exit" : "navigation/fullscreen"
+                        onTriggered: {
+                            toggleFullScreen()
+                        }
```

Comment:
Can we move this to the right of the refresh button?
Also can we hide this button in normal mode, showing it only in fullscreen mode?
I think we should also have an item in the menu (which is always shown).
That way we are consistent with the download action which has a menu item but is only shown in the toolbar when there is an active download.